frontend/actions/index.js

```diff
@@ -4,11 +4,11 @@ import { nanoid } from 'nanoid';
 
 const frontendId = nanoid(16);
 
-let isMounted = false;
+let isMounted = false, tempSocket = null;
 
 const mountSockets = (store) => {
     if (isMounted) {
-        return;
+        return tempSocket;
```

Comment:
What is `tempSocket`? 🤔

Reply:
If we open the run page and come back to home, it was previously returning nothing therefore the sockets weren't working. This is just a copy of the actual socket so we can return it when another page calls `_app.js`